portfolio/src/main/webapp/script.js

```diff
@@ -12,6 +12,28 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+var slideIndex = 0;
+showSlides();
```

Comment:
since the slides aren't added to the document yet at the time this line executes, it's not actually doing anything, and can be removed

Reply:
I took out showSlides() but I think I still need slideIndex= 0 because otherwise slideIndex isn't declared so there is nothing to increment in the function -  I tried removing it or moving it to inside the function and it didn't work.